packages/react-devtools-shared/src/backend/agent.js

```diff
@@ -132,6 +132,12 @@ type PersistedSelection = {|
   path: Array<PathFrame>,
 |};
 
+type LaunchEditorParams = {|
+  launchEditorEndpoint: string,
+  fileName: string,
+  lineNumber: string,
+|};
```

Comment:
Question: What do the | do around the enums?  Never seen that before?

Reply:
Just do same as other type definintion around the file, maybe it's `@flow` 's grammer, I am not sure. I don't hava enough experience abhout `@flow` to explain it.( https://flow.org/en/docs/types/unions/
![2021-02-16 10 51 46](https://user-images.githubusercontent.com/14012511/108012689-fa0a5400-7044-11eb-9c25-d7374a64e815.png)